regolith/tools.py

```diff
@@ -2104,39 +2105,47 @@ def google_cal_auth_flow():
         token.write(creds.to_json())
     # Save the credentials for the next run
 
-def add_to_gitlab(name):
-    """Takes a newly created event, and adds it to the user's google calendar
+def create_talk_repo(name, rc):
```

Comment:
in principle this can be used to create any repo, not just a talk repo, right?  For example Jay has a project to create a repo for a prum.  I suggest change the name to `create_repo`